src/scss/grommet-core/_objects.article.scss

```diff
@@ -105,3 +117,16 @@
 .grommet article:not(.article) {
   width: 100%;
 }
+
+.article__pagination {
+  @include media-query(lap-and-up) {
+    display: none;
+  }
+
+  @include media-query(palm) {
+    position: fixed;
```

Comment:
I thought we would be rendering at the bottom of the page without positioning fixed.